ds_judgements_public_ui/templates/includes/facets/year_search_facets.html

```diff
@@ -0,0 +1,8 @@
+{% load query_filters court_utils %}
+<div style="text-align: left;">
+  <b>Years relevant to your query:</b>
+  {% for year, item_count in context.year_facets.items %}
+    <a href="{% url "search" %}?{{ context.query_params|replace_year_in_query:year }}">{{ year }} ({{ item_count }})</a>
```

Comment:
This feels a lot cleaner than court_search_facets.

Reply:
I assume it gives things in a nice useful format.

Reply:
The assumption was wrong at the time, hence the live demo fail 😅 . But we've fixed it now! :D